\section{Protein Synthesis}
{\color{red} CITATIONS PLEASE}

%This section aims to provide a basic introduction to the biology of protein synthesis. 
%DNA is a long macromolecule consisting of a sugar phosphate backbone with attached with nucleotide bases Guanine, Cytosine, Adenine and Thymine (abbreviated as G, C, A,  and T) (CITATION)

%Yo, so DNA is like a double stranded, long stringy boy in a double helix thing with some nucleotide stuff in the middle. It's long, like really long. Proportionally longer than Jono long but that's okay because it's really tightly wound. Instead of having one super long DNA string with everything in it, the genome is divided up into these chromosome bois and they're conveniently stored in the nucleus (in cells that have them).

%eople were kinda unsure about how DNA looked for ages, but then some white dudes did some Xray chrystallography and some math and argued with some other white dudes and we ended up with the molecular structure that we have now. 

%In the DNA there are these bits that are kinda important because they help make proteins and proteins are kinda important. These important bits are called genes. All cells hold the same genome in a given organism but they all express different genes so that the different cell types can do different stuff. There are some caveats here about the fact that even though there's alike a bazillion nucleotides in the DNA of an organism, only a teensy bit isn't JUNK. 

%To make a protein, some squishy lads (TF \& RNA Polymerase) come along and bind near the transcription start site and get going. This makes some RNA. RNA is like DNA, except halved so it's got only one backbone (that's made of NOT sugar-phosphate) and one of the nucleotides is a U instead of a T. 

%Then the RNAs get out of the nucleus and they go to 
%LOL JOKE, turns out most of the junk bits actually help with the initiation and regulation of this fancy transcription process. There are heaps of tricky models for how enhaceosomes wotk but the main point is that they're out there.

The mechanisms underlying the transfer of genetic information was of great interest to scientists in the 20th century.
Experiments performed by Avery, McCleod and McCarty in 1944 demonstrated the molecule deoxyribonucleic acid (DNA) as the carrier for genetic information~\cite{macleod1944studies}. While this had provided evidence for the existence of some molecule of heredity, it was not until Watson and Crick~\cite{watson1953structure} performed X-ray crystallography experiments and suggested the now familiar structure for DNA that provides the basis for heredity mechanics. DNA is a long molecule consisting of a sugar phosphate backbone chains arranged in a double helix held together by pairs of nucleotide bases: cystine (C), guanine (G), adenine (A) and thymine (T). It was Crick~\cite{crick1958protein} that proposed one of the most enduring ideas of molecular biology; the \emph{Central Dogma}. Simply, the Central Dogma says that DNA is transcribed to make ribonucleic acid (RNA); which are then translated to proteins. 
The structure of DNA is the pivotal underlying idea behind the central dogma.


Proteins play an important since they are required for the repair, maintenance and growth of an organism. 
A gene is a region in the DNA that contains the necessary information for the synthesis of an associated protein. In eukaryotes, genes are embedded in DNA and collected in tightly packed configurations in the nucleus of cells ~\cite{alberts2002chromosomal}. Typically, genes are flanked by regulatory regions that act as binding sites for enzymes to assist in the initiation and regulation of gene expression.
All the nucleated cells in an organism contain the same DNA but different specialized cell types each each express a different profile of genes owing to their different roles and function~\cite{lockhart2000genomics}. 
%A protein complex known as chromatin helps in the storage of DNA by wrapping 

%The process of transcribing deoxyribonucleic Acid (DNA) to ribonucleic Acid (RNA) and the subsequent translation from RNA to protein is fundamental to the study of molecular biology. So much so that this process, first suggested by Crick, is often widely known as the central dogma of molecular biology \cite{crick1958protein}. Below is a brief summary of the relevant molecules, proteins and enzymes involved in the process. 

According to the central dogma, protein synthesis occurs in two stages; the first being the transcription of DNA to RNA. 
During the first stage, proteins called transcription factors bind to regulatory regions surrounding the gene to recruit RNA polymerase; an enzyme responsible for the synthesis of a sub type of RNA known as messenger RNA (mRNA). Note that these regulatory regions are not transcribed in this process. 
The mRNA is able to leave the nucleus for further processing, due to the smaller size of the molecule. 

However, not all of the sequence encoded in the DNA, and subsequently RNA, is required for protein synthesis. Translated regions are further subdivided in to introns and exons. Splicing is a process in which enzymes cut the introns out of the gene sequence and join the exons (see fig~\ref{fig:splicing}).

    \begin{figure}
        \centering
        {\color{red} \textbf{insert diagram of splicing}}
        %\includegraphics{}
        \caption{The processing of transcripted sequence removing the intronic regions, making the final mRNA sequence}
        \label{fig:splicing}
    \end{figure}
    

    

The second phase, known as translation, is primarily carried out by a molecule called a ribosome and by translational RNA (tRNA) outside of the nuclues. These tRNAs are another sub-type of RNA that can bind to amino acids. The ribosomes bind to the spliced mRNA sequences and are responsible for recruiting tRNAs, {\color{red}unloading} their amino acids and forming amino acid chains. The complete chain of amino acids is (in essence) the {\color{red}final} product, known as a protein. 

\subsection{Non-coding things}
There are some competing schools of thought regarding the changes of gene expression; the main two being genetics and epigenetics. While both fields are concerned with the study of changes in gene function and heredity, the main motivating ideas behind both are fundamentally different. Genetics is concerned with the change of expression brought about through changes in the genetic sequence. In contrast, epigenetics is concerned with the change of gene expression and function by external factors from the actual genetic material itself, such as with histone modification. The work in this thesis will focus on things from a genetics perspective. As such, a common way to change the expression of genes is through the change of surrounding regulatory regions.

These enhancers are regions flanking the genes and contain binding sites for proteins (transcription factor) that regulate the expression of corresponding genes. There exist different models for the ways that these enhancers work for the TFs; a complete review may be found in {\color{red} CITATION}. In summary, there are like three and they all behave slightly differently for protein-protein dynamics and stuff.

\subsection{Epigenetics}
    WHAT IS A THING
        \begin{itemize}
            \item Changes in the environment can change phenotype. 
        \end{itemize}
\subsection{Enhancers}
	DNA structure consists of four nucleic acids: Guanine, Cytosine, Adenine and Thymine (abbreviated as G, C, A,  and T) attached to a sugar phosphate back bone.
	The {\color{red}entirety of the genetic material (What about E P I G E N E T I C S)} in eukaryotic organism is coded as a long sequence of these nucleotides. Initial estimates for the protein coding proportion of the genome was placed at  2\%~\cite{international2001initial}. 
	The remainder of the non-coding regions were originally though to be \emph{junk} DNA. This thinking has progressively changed over time and non-coding regions have been shown to be locations for transcription factor binding sites for the assistance in the regulation of gene expression. {\color{red} The ENCODE project estimates that 80\% of the genome is functional, far greater than $<$2\% coding for proteins \textbf{citation}}
	It has been theorized that the conservation of these non-coding regions throughout evolutionary distances is some indication of function.
	
	Regions surrounding important genes such as the SOX2 gene have been shown to show a local neighbourhood lacking in other genes but heavily populated by conserved enhancer regions accross species {~\cite{uchikawa2003functional}}. In fact, these conserved non-coding elements have been found across independent vertebrate organisms. This is less so when comparing vertebrate with invertebrate species. 
	
%	Previously, the remainder of the non-coding regions were thought to be \emph{junk} DNA. This thinking has progressively changed over time and non-coding regions have been shown to be locations for transcription factor binding sites for the assistance in the regulation of gene expression. 
	
	Protein coding genes actually only make up a small portion of the genome. Estimates show that only approximately 2\% of the genome is protein coding, but a much larger 80\% is functional \textbf{CITE}. Enhancers are a subset of these non-coding regions that act as binding sites for proteins such as transcription factors and play a role in the regulation of protein synthesis.
	
	There are some competing models on the mechanisms behind enhancers. In short, the three main models are the enhanceosome, the billboard model and the TF collective; each of these suggest different mechanics and dynamics between TF-DNA and TF-TF pairings as well as the modes of activation for the enhancer. For a detailed review see Spitz and Furlong ~\cite{spitz2012transcription}. These enhancer regions are typically found flanking coding regions of DNA. Previous work by Algama~\cite{algama2017genome} have used statistical methods detailed in later sections to identify these non-coding functional elements.
	
% 	\subsection{Spitz and Furlong}
% 	{\color{red} Rework this section to fit in with Enhancers}
% %	Let us preface this section with this idea that nothing is ever definite in biology and that in general, it depends. 
% 	Enhancer regions are part of the subset of DNA that is considered to be non-coding. 
% 	Some of these enhancer regions can be sites for transcription factor binding and are thought to play important roles in the process of gene expression. 
% 	There two competing models proposed about how these enhancer regions work in the recruitment and attraction of transcription factors and how they encourage binding. 
% 	That being said, the general consensus seems to be that there is no global rule that determines the nature of the binding of transcription factors to these regions. 
% 	 {\color{red} TALK ABOUT DIFFERENT MODELS.}
	 
% 	 {\color{blue}Also, there have been some hypothesis about the importance of these regulatory modules in the developmental phase of organisms. It is thought that the programmed regulation of certain genes might be the cause for phenotypical differences among species. One notable example of the is the presence of the same family of genes being present across organisms spanned by a large evolutionary distance in the cardiac genes leading to differences in the amount of heart chambers that are present. }{\color{red}There's no citation. MIRANAS STUDY?}
	
	
% 	{\color{green}What is even cooler and might add evidence towards the fact that these regulatory regions are so important is that these non-coding elements in the DNA are very slow evolutionarily. So, like, that means they must be really important for all organisms, right? Another thing that's really  cool is that there are these regions around genes called gene deserts and the presence of certain proteins around these regions drops off really dramatically. Now, this might mean that there are inhibitors and stuff around the place that make sure that only the gene relevant to the proteins is being properly maintained.  }{\color{red} this is sort of up there}