\section{Enhancers}
    Transcription factors are important because they modify the transcriptional machinery
    
    How do they do it?
        Gene expression is crucial to the function and differentiation of different cell types and required for growth, repair and maintenance of an organism.  In eukaryotic cells, genes are embedded in DNA and collected in tightly packed configurations, known as chromatin, in the nucleus of cells~\cite{alberts2002chromosomal}.  A gene is a region in the DNA that contains the necessary information for the synthesis of an associated protein.
        The types of genes and the intensity associated with their expression in a cell is known as an expression profile. All the nucleated cells in an organism contain the same DNA, but the differences in their function arises from the different expression profiles~\cite{lockhart2000genomics}. Our interest lies in the identification and classification of certain regions in the DNA that help regulate the expression of particular genes. 
       

        One of the enduring tenets of molecular biology is that gene expression occurs as a result of DNA being transcribed to RNA and then translated to proteins~\cite{crick1958protein}.During the first stage, proteins called transcription factors (TFs) bind to regulatory regions surrounding the gene to recruit RNA polymerase; an enzyme responsible for the synthesis of a sub type of RNA known as messenger RNA (mRNA). 
        The second phase, known as translation, is primarily carried out by a molecule called a ribosome and by translational RNA (tRNA) outside of the nucleus. The end result is a protein comprised of an amino acid chain.
        
        The process is highly regulated at many levels with the involvement of a large number of factors with a variety of roles~\cite{lemon2000orchestrated}. The main mechanism by which TFs interact with DNA, and subsequently regulate transcription, is through the binding of TFs to small sequences in the DNA. the In this work, we focus on transcriptional regulation, through enhancers.
        
        Changes in gene expression can often be attributed to changes in either the direct genetic sequence (\emph{eg.} mutation), or epigenetic factors in the expression process. There are numerous ways that epigenetic factors can alter the expression of genes without changes to the genetic code itself. A readily available example of an epigenetic factor are TFs. These TFs take on a diverse set of roles in regulating transcription, such as the initiation of the process and the recruitment of other factors~\cite{lemon2000orchestrated}.
        
        
        Conservation of sequence as a single metric has been used for the identification of enhancers. This approach has proven useful in the past, but in the context of cardiac enhancers, relatively few have been found. Sequencing has been used, in conjunction with knowledge about binding proteins~\cite{blow2010chip}
        
        Enhancers may show conservation of function across a large evolutionary distance, even if the sequences have diverged significantly~\cite{tautz2000evolution}
        
        That doesn't mean that there is no loss of sequence conservation. It has been shown in tissue specific studies that gene expression is much higher conserved than TF binding. Positive correlation with combinatorial TF binding and transctipion levels of closeby genes~\cite{wong2014decoupling}
        
        We have some data in the form of ChIP-Seq h3K4me1. This denotes the methylation status of a histone. This particular site often associated with promoter regions of actively being transcribed genes~\cite{barski2007high}. 
        %https://epigenie.com/key-epigenetic-players/histone-proteins-and-modifications/histone-h3k4/
        While this association has been used to identify active and primed enhancers, it is poorly understood whether H3K4me1 influences, is influenced by or correlates enhancer activity~\cite{rada2018h3k4me1}
        
        The many faces of histone lysine methylation 2002)
        
        Revisit Spitz and Furlong to see what sort of idease we have to give an overlay of what a TF is and how we have different models for regulation. 
        
       
        
        
        
        As an example, the remodelling of chromatin through histone methylation can alter the accessibility of transcriptional machinery to reach their target genes~\cite{gibney2010epigenetics, holoch2015rna }. 
        
        % Changes in gene expression is primarily driven by direct changes in the sequence (genetic) or changes in epigenetic factors involved in the expression process, such as chrmomatin remodelling through histone methylation altering accessibility between transcriptional machinery and their target gene~\cite{gibney2010epigenetics, holoch2015rna }.% (RNA CAN ALSO REGULATE GENE EXPRESSION MAYBE) 
        Epigenetics is important~\cite{holliday2006epigenetics}
        DNA Methylation is super important when considering stability of gene epxression~\cite{jaenisch2003epigenetic}
        
        Comparative genomics might not be the best way since these things display modest or no conservation. Sequence conservation does not provide clues to function, even if it is identified as an enhancer~\cite{pennacchio2013enhancers}
        
        
        We have many different 